src/components/DropdownItem.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { CheckIcon } from './icons';
+
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+
+const DropdownItem = ({
+  title,
+  description,
+  Icon,
+  className,
+  active,
+  onClick,
+  href,
+  target,
+  children,
+  disabled,
+  noItemsActiveState,
+}) => {
+  const titleText = typeof title === 'string' ? title : title.text;
+  const ActionElem = href ? 'a' : 'button';
+
+  return (
+    <div
+      className={classnames(
+        'drui-dropdownItem',
+        {
+          [className]: className,
+          ['drui-dropdownItem--withDescription']: description,
+          ['drui-dropdownItem--active']: active,
+          ['drui-dropdownItem--disabled']: disabled,
+          ['drui-dropdownItem--noItemsActiveState']: noItemsActiveState,
+        })
+      }
+      onClick={onClick}
+      key={`drui-dropdown-item-${titleText.split(' ').join('').toLowerCase()}`}
+    >
+      <ActionElem
+        href={href || null}
+        type={!href ? 'button' : null}
+        className="drui-dropdownItem__action"
+        target={target || null}
```

Comment:
I think we can just do `target={target || null}`

Reply:
Hmmm... Not sure I get it. It seems to me that this is exactly how it's done :)
`target={target || null}`